src/types.rs

```diff
@@ -223,6 +223,27 @@ pub struct Dependencies {
     pub dependencies: Vec<Dependency>,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ReverseDependency {
+  pub crate_version: Version,
+  pub dependency: Dependency,
+}
+
+// This is how reverse dependencies are received
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ReverseDependenciesAsReceived {
```

Comment:
This should not be public.
It can be `pub(super)` instead.

Reply:
Much better this way indeed. I didn't know that was an option.